src/index.html

```diff
@@ -5,10 +5,82 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+      href="#"
+      class="header__logo-link">
+        <img
+        src="./images/logo.png"
+        alt="logo"
+        width="40"
+        height="40" >
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+            href="#"
+            class="nav__item-link is-active">
+            Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+            href="#"
+            class="nav__item-link">
+            Samsung
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a
              href="#"
              class="nav__item-link is-active"
            >
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a
              href="#"
              class="nav__item-link"
            >
              Samsung
            </a>
          </li>
```
please fix indentation and code style in all places